gramineproject/gramine
# [PAL/Linux-SGX] Add missing fields in local attestation

## Description of the changes 
See the commits.

## How to test this PR? 
You need a machine which reports `1` (or rather lowest bit set) in supported miscselect (you can use `is-sgx-available` from https://github.com/gramineproject/gramine/pull/1075 PR to check). Then try enabling `exinfo` in manifest - without this PR local attestation will fail.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/1074)
